Readme.md

```diff
@@ -191,6 +192,57 @@ while (keys.length) {
 
 [crockfordconvention]: http://javascript.crockford.com/code.html
 
+## Declare Classes
+
+Declare a class using the `@constructor` jsdoc annotation. Like,
+
+```js
+/**
+ * @constructor
+ */
+function SomeClass() {}
+```
+
+While there are several ways to attach methods and properties to an object
+created via `new`, the preferred style for methods is:
+
+```js
+SomeClass.prototype.someMethod = function() {
+  /* ... */
+};
+```
+
+The preferred style for other properties is to initialize the field in the
+constructor:
```

Comment:
This is nice. but put a note that the property will be instance local and not prototype local.